### DISCUSSION TITLE
Custom component in sidebar not showing on menu (mobile sidebar)

### DISCUSSION THREAD

**Discussion by glitch-txs** [facebook/docusaurus#10197] [Q&A]
Hi! I have created a custom component in the sidebar to select different frameworks, but when the sidebar changes to a menu on mobile devices this component disappears. How can I have more control over this menu?

Also thanks Slorber for being so active answering questions 🙏 

![image](https://github.com/facebook/docusaurus/assets/66949816/74a25236-f4c2-4540-9a99-bbf016497148)
![image](https://github.com/facebook/docusaurus/assets/66949816/4b02a2a2-2874-43cb-b018-3418d3e8f024)

This is what I'm currently using

```ts
import DocSidebar from '@theme-original/DocSidebar'
import FrameworksMenu from '../FrameworksMenu'
import '../../css/sidebar.css'

export default function DocSidebarWrapper(props) {
  return (
    <>
      <FrameworksMenu />
      <DocSidebar {...props} />
    </>
  )
}
```

**Comment by glitch-txs** [comment]
resolved 👍

**Comment by htessaro** [comment]
Hi @glitch-txs . How did you add a select into the sidebar?

**Reply by glitch-txs** [reply]
I had to do it manually 

You can check the source code here
https://github.com/WalletConnect/walletconnect-docs
